==== build_tools/run_after_chdir.py#72 - build_tools/run_after_chdir.py ====
--- build_tools/run_after_chdir.py#72	(revision 72)
+++ build_tools/run_after_chdir.py	(working copy)
@@ -58,7 +58,7 @@
   # We don't capture stdout and stderr from Popen. The output will just
   # be emitted to a terminal or console.
   print sys.argv
-  process = subprocess.Popen(sys.argv)
+  sys.exit(subprocess.call(sys.argv))
 
 if __name__ == '__main__':
   main()
